build/board/Sabre_iMX6SX_1GB/Sabre_iMX6SX_1GB_FMFileList.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FMCollectionManifest xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns="http://schemas.microsoft.com/embedded/2004/10/ImageUpdate" Product="IoTUAP" >
+  <IsBuildFeatureEnabled>true</IsBuildFeatureEnabled>
+  <FMs>
+    <FM ID="Sabre6Q" Path="$(FMDirectory)\Sabre_iMX6SX_1GB_DeviceFM.xml" ReleaseType="Production" OwnerName="NXP" OwnerType="OEM" CPUType="ARM" Critical="true"/>
```

Comment:
Looks like this is a bug in my newboard script. Its not critical, it just changes the name of some temporary files, but might as well stay consistent.
```suggestion
    <FM ID="Sabre6SX" Path="$(FMDirectory)\Sabre_iMX6SX_1GB_DeviceFM.xml" ReleaseType="Production" OwnerName="NXP" OwnerType="OEM" CPUType="ARM" Critical="true"/>
```

Reply:
Fixed.